modules/ROOT/pages/scopes-and-collections-config.adoc

```diff
@@ -0,0 +1,191 @@
+                function(doc, oldDoc, meta) {
+                    if (doc.type != "default") {
+                        throw({forbidden : "Rejected document"})
+                    }
+                    channel("legacy")
+                }
+            `,
+            "import_filter": `
+              function(doc) {
+                return doc.type == "mobile"
+              }
+            `
+        },
+        "collection2" : {
+          "sync": `
+            function(doc, oldDoc, meta) {
+              channel("collection1")
+            }
+          `,
+          "import_filter": `
+              function(doc) {
+                return doc.type == "mobile"
+              }
+          `
+        }
+      }
+    }
+  },
+  "num_index_replicas": 0
+}
+----
+
+== Collections Configuration Description
+
+Here is the description for the custom `Scope` with 2 custom `Collections` configuration. 
+The database's structure, collections, sync functions, import filters, and index replicas are crucial in accurately validating, filtering, and synchronizing documents within the database. 
+
+== Database Information
+
+=== db
+This sets the name of the database to "db".
+
+=== bucket
+This indicates the bucket's name where the data will be stored, in this case, "bucket".
+
+== Scopes
+A Scope can contain a set of Collections. 
+You have the option to use the default provided by Couchbase or customize it to fit your specific needs.
+
+=== scope1
+
+The "scopes" field is not required for upgrading to Sync Gateway 3.1. 
+Even if this field is not specified in the configuration, the previous versions of the config file are still compatible with the new version.
+Therefore, the provided examples are not essential for upgrading to Sync Gateway's latest version.
+
+== Collections
+A Collection can contain a set of documents. 
+You have the option to use the default provided by Couchbase or customize it to fit your specific needs.
+
+=== Collection1
+
+=== Sync Function:
+[source,javascript]
+----
+function(doc, oldDoc, meta) {
+    if (doc.type != "default") {
+        throw({forbidden : "Rejected document"});
+    }
+    channel("legacy");
+}
+----
+While the example sync function here has a specific purpose, it's crucial to understand that sync functions can perform various actions.
+
+Whenever a document is synced or updated in collection1, a function called `sync` is executed. 
+This function has three parameters: the new document (doc), the existing document (oldDoc), and document metadata (meta). 
+If the type field of the document is not `default`, the function will throw an error message saying "Rejected document" and call the `channel` function with the argument "legacy".
+
+=== Import Filter:
+[source,javascript]
+----
+function(doc) {
+    return doc.type == "mobile";
+}
+----
+When importing documents into collection1, there's an import filter that checks the `type` field value to determine if a document should be imported. 
```

Comment:
When importing documents into collection 1, an import filter checks the 'type' field...